pages/[category]/index.tsx

```diff
@@ -1,56 +1,91 @@
+import ArrowIcon from 'assets/icons/ArrowIcon'
+import BreadCrumbs from 'components/Breadcrumps'
+import MessageIcon from 'assets/icons/MessageIcon'
 
 const CategoryPage = () => {
   const router = useRouter()
-  const { category } = router.query as { category: string}
+  const { category } = router.query as { category: string }
 
-  const subcategories: SubCategories[] = []
-  sidebarData.forEach((c) => {
-    if (c.category == category)
-      c.subcategory.forEach((a) => {
-        subcategories.push(a)
-      })
-  })
+  const subcategories: SubCategories[] = category
+    ? sidebarData.filter((c) => c.category == category)[0]['subcategory']
+    : []
 
   return (
     <section className="flex min-h-[calc(100vh-165px)] flex-col">
       <div className=" m-auto items-start gap-2">
         {subcategories?.length ? (
           <>
-            <div className="m-auto gap-2 flex flex-col items-center justify-center ">
-              <p className="md:text-4xl text-xl uppercase font-bold text-theme-secondary dark:text-gray-text w-fit mx-auto text-center">
-                <span className="text-theme-primary">-/</span>
-                {category.split('-').join(' ')}
-              </p>
-              <p className="text-xl hidden md:block">
-                Get access to all exclusive{' '}
-                <span className="capitalize text-theme-primary">
-                  {category.split('-').join(' ')}
-                </span>{' '}
-                resources!
-              </p>
+            <div className="flex mt-8 justify-between	">
+              <BreadCrumbs
+                sections={[{ name: category, url: `/${category}` }]}
+              />
+              <div className="flex my-auto gap-x-3">
+                <MessageIcon />
+                <div className="text-md text-text-quinary hidden md:block">
+                  Give a feedback
+                </div>
+              </div>
             </div>
-            <div className="flex flex-wrap gap-3 mt-6 justify-center items-start mx-auto lg:max-w-sm max-md:max-w-sm">
-              {subcategories.map((subcat, i) => (
+            <div className="flex max-w-1038 justify-center">
+              <div className="flex flex-wrap gap-x-9 gap-y-5 mt-6 mb-28 flex-start items-start">
+                {subcategories.map((subcat, i) => (
+                  <Link
+                    key={i}
+                    href={`/${category}${subcat.url}`}
+                    aria-label={`Explore ${subcat.name}`}
+                    className={`group flex border dark:border-theme-primary dark:border-opacity-8 rounded-xl bg-theme-tertiary  hover:bg-theme-quaternary hover:bg-opacity-25	 px-3 py-1 dark:hover:border-opacity-25 hover:text-dark-primary shadow-lg lg:max-w-xs max-md:max-w-xs px-4 py-4 group`}
+                  >
+                    <div>
+                      <div
+                        className={`dark:text-text-primary text-lg capitalize font-semibold	`}
```

Comment:
Please, use template literals only when it's required and avoid extra spaces.

```suggestion
                        className="dark:text-text-primary text-lg capitalize font-semibold"
```